pages/auth/authorize.vue

```diff
@@ -20,15 +20,28 @@
         </div>
       </div>
       <div class="title">
-        <h1>Authorize {{ app.name }}</h1>
+        <h1>{{ formatMessage(messages.authorizeAppNameLabel, { app_name: app.name }) }}</h1>
       </div>
       <div class="auth-info">
         <div class="scope-heading">
-          <strong>{{ app.name }}</strong> by
-          <nuxt-link class="text-link" :to="'/user/' + createdBy.id">{{
-            createdBy.username
-          }}</nuxt-link>
-          will be able to:
+          <IntlFormatted
+            :message-id="messages.appInfoLabel"
+            :values="{
+              app_name: app.name,
```

Comment:
Any particular reason to use snake_case instead of camelCase?

Reply:
Not really. Indeed, it will be better with camelCase ^^'.

Reply:
Fixed ^^ !